### DISCUSSION TITLE
How to delete a keymap?

### DISCUSSION THREAD

**Discussion by Aman9das** [quarto-dev/quarto-nvim#116] [Q&A] [answered]
```lua
  {
    "quarto-dev/quarto-nvim",
    opts = {
      lspFeatures = {
        languages = { "r", "python", "julia", "bash", "html", "lua" },
      },
      codeRunner = {
        enabled = true,
        default_method = "slime", -- "molten" or "slime"
      },
      keymap = {
        rename = "<leader>lG",
        format = nil,
      }
    },
  }
```

I can change the `rename` keymap but the `format` keymap isn't getting deleted. Am new to Neovim.

Similar to: https://github.com/jmbuhr/otter.nvim/issues/71

**Comment by jmbuhr** [comment, marked as answer]
I made some changes and now you can disable all keymaps by setting `keymap` to `false`, or disable individual ones by setting them to `false`. Setting the `nil` will not disable it but rather use the default due to the way I merge the default with the user config.
